tests/tests/regression_tests.rs

```diff
@@ -136,13 +142,10 @@ fn trial_for_test(opts: &RuffleTestOpts, test: Test, list_only: bool) -> Trial {
             }
         }
     });
-    if ignore {
-        trial = trial.with_ignored_flag(true);
-    }
-    if let Some(name) = subtest_name {
+
+    trial
+        .with_ignored_flag(ignore)
         // Put the subtest name as the test 'kind' instead of appending it to the test name,
         // to not break `cargo test some/test -- --exact` and `cargo test -- --list`.
```

Comment:
After renaming `subtest_name` to `test_kind` it makes sense to move it over the line that adds the subtestname to the test kind.